src/components/HearingList.js

```diff
@@ -113,19 +113,19 @@ HearingListItem.propTypes = {
 class HearingList extends React.Component {
 
   render() {
-    const {hearings, isLoading, labels, handleSort, handleSearch, handleLabelSearch, language, handleChangeTab, activeTab, showOnlyOpen, toggleShowOnlyOpen} = this.props;
+    const {hearings, isLoading, labels, handleSort, handleSearch, handleLabelSearch, language, activeTab, showOnlyOpen, toggleShowOnlyOpen} = this.props;
     const hearingsToShow = !showOnlyOpen ? hearings : hearings.filter((hearing) => !hearing.closed);
     const hearingListMap = (hearingsToShow ? (<div className="hearing-list-map map">
       <Checkbox inline readOnly checked={showOnlyOpen} onChange={() => toggleShowOnlyOpen()} style={{marginBottom: 10}}>
         <FormattedMessage id="showOnlyOpen"/>
       </Checkbox>
-      <OverviewMap hearings={hearingsToShow} style={{width: '100%', height: '40%'}} enablePopups />
+      <OverviewMap hearings={hearingsToShow} style={{width: '130%', height: 600}} enablePopups />
```

Comment:
Hm, now the map flows over the edge on small devices. Is it possible to get the width fixed to the screen size (not viewport size) so it would span the screen width? Or would we need to actually make the whole list view (tabs and all) fit the screen width?